indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/models/Credential.kt

```diff
@@ -93,12 +94,17 @@ data class Credential(
     @JsonProperty("rev_reg") val revocationRegistry: RawJsonMap?,
     val witness: RawJsonMap?,
     @JsonProperty("rev_reg_id") override val revocationRegistryIdRaw: String?,
-    val values: Map<String, CredentialValue>,
+    val values: CredentialProposal,
     val signature: Map<String, RawJsonMap?>,
     val signatureCorrectnessProof: RawJsonMap
 ) : ContainsSchemaId, ContainsCredentialDefinitionId, ContainsRevocationRegistryId
 
-data class CredentialValue(val raw: String, val encoded: String)
+data class CredentialValue(
+    val raw: String,
+    val encoded: String = if (raw.toLongOrNull() == null)
+        BigInteger(raw.toByteArray(StandardCharsets.UTF_8)).toString(10)
+    else raw
+)
 
 data class CredentialInfo(val credential: Credential, val credRevocId: String?, val revocRegDeltaJson: String?)
```

Comment:
Same with com.luxoft.blockchainlab.hyperledger.indy.models.CredentialReference?

Reply:
I don't get it.

Reply:
Same dataclass